crates/uv/tests/workspace.rs

```diff
@@ -0,0 +1,367 @@
+    "###
+    );
+
+    context.assert_file(current_dir.join("check_installed_bird_feeder.py"));
+    uv_snapshot!(context.filters(), install_workspace(&context).arg(&current_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Audited 1 package in [TIME]
+    "###
+    );
+
+    context.assert_file(current_dir.join("check_installed_bird_feeder.py"));
+}
+
+#[test]
+fn test_albatross_in_examples() {
+    let context = TestContext::new("3.12");
+    let current_dir = workspaces_dir().join("albatross-in-example");
+
+    uv_snapshot!(context.filters(), install_workspace(&context).arg(&current_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Built 1 editable in [TIME]
+    Resolved 2 packages in [TIME]
+    Downloaded 1 package in [TIME]
+    Installed 2 packages in [TIME]
+     + albatross==0.1.0 (from file://[WORKSPACE]/scripts/workspaces/albatross-in-example)
+     + tqdm==4.66.4
+    "###
+    );
+
+    context.assert_file(current_dir.join("check_installed_albatross.py"));
+    uv_snapshot!(context.filters(), install_workspace(&context).arg(&current_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Audited 1 package in [TIME]
+    "###
+    );
+
+    context.assert_file(current_dir.join("check_installed_albatross.py"));
+}
+
+#[test]
+fn test_albatross_just_project() {
+    let context = TestContext::new("3.12");
+    let current_dir = workspaces_dir().join("albatross-just-project");
+
+    uv_snapshot!(context.filters(), install_workspace(&context).arg(&current_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Built 1 editable in [TIME]
+    Resolved 2 packages in [TIME]
+    Downloaded 1 package in [TIME]
+    Installed 2 packages in [TIME]
+     + albatross==0.1.0 (from file://[WORKSPACE]/scripts/workspaces/albatross-just-project)
+     + tqdm==4.66.4
+    "###
+    );
+
+    context.assert_file(current_dir.join("check_installed_albatross.py"));
+    uv_snapshot!(context.filters(), install_workspace(&context).arg(&current_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Audited 1 package in [TIME]
+    "###
+    );
+
+    context.assert_file(current_dir.join("check_installed_albatross.py"));
+}
+
+#[test]
+fn test_albatross_project_in_excluded() {
+    let context = TestContext::new("3.12");
+    let current_dir = workspaces_dir()
+        .join("albatross-project-in-excluded")
+        .join("excluded")
+        .join("bird-feeder");
+
+    uv_snapshot!(context.filters(), install_workspace(&context).arg(&current_dir), @r###"
+    success: true
+    exit_code: 0
+    ----- stdout -----
+
+    ----- stderr -----
+    Built 1 editable in [TIME]
+    Resolved 4 packages in [TIME]
+    Downloaded 3 packages in [TIME]
+    Installed 4 packages in [TIME]
+     + anyio==4.3.0
```

Comment:
Can we just come up with an example that uses fewer, smaller packages? E.g., do we really need boltons, which is almost 200 KB?